treesapp_assign_and_update.Rmd

```diff
@@ -27,47 +41,45 @@ treesapp assign \
   --output p_amoA_FunGene9.5_isolates_assign/
 ```
 
-The command should take a few seconds to run and if you see a `TreeSAPP has finished successfully.` then you're good to move on.
+The command should take a few seconds to run and if you see a 'TreeSAPP has finished successfully.' at the end then you're good to move on.
 
 ## Analyze the classifications
 
-### iTOL
+### The interactive Tree of Life
 
-First off, we can quickly and easily view where the sequences were placed on the phylogeny using the interactive Tree of Life (iTOL) [@Letunic.2019]. Navigate to <https://itol.embl.de/> using a web browser. Open the directory `p_amoA_FunGene9.5_isolates_assign/iTOL_output/XmoA/` and upload the file XmoA_complete_profile.jplace. This JPlace file contains both the phylogeny and the location of the placements that will be visualized.\
-Next, navigate to the page displaying the phylogeny and click-and-drag the file `XmoA_labels.txt` into the iTOL window. This should convert the TreeSAPP identifiers (e.g. 1_XmoA) to more useful descriptions with the organism name and accession for each leaf node.
+Before proceeding to update the reference package we should view where the sequences were placed on the phylogeny. By just looking at where the query sequences were placed you can get a good idea of how closely related they are to the sequences in the reference package.
+We will again use the interactive Tree of Life (iTOL) to visualize the JPlace file that is produced by TreeSAPP [@Letunic.2019].
 
-Finally, turn on the "Phylogenetic Placements" dataset (right of the screen) and the figure should look identical to this figure on the right.
+You will need to copy the files to be uploaded to iTOL from the server to your local computer (i.e. laptop or desktop) using `scp`. From your local computer, run the following command to copy the 'iTOL_output' directory in the TreeSAPP outputs to your Desktop.
 
--   the figure should look identical to Figure \@ref(fig:XmoA-FunGene).
-
-![(\#fig:XmoA-FunGene)The XmoA phylogeny with FunGene sequences placed onto it. The red bubbles represent sequence placements with the size of the bubble indicating the number of placements on a branch. There were 74 sequences classified, 34 of which were placed on the clade representing members of the *Gammaproteobacteria* order *Methylococcales*. Another 15 sequences were placed on the clade of *Betaproteobacteria*, represented by the genera *Nitrosospira* and *Nitrosomonas*.](images/P_AmoA_tutorial_iTOL.png)
+```
+scp root@<ip address>:/data/ts_tutorial/p_amoA_FunGene9.5_isolates_assign/iTOL_output ~/Desktop/
+```
```

Comment:
````suggestion
```{bash eval = FALSE}
scp - r root@<ip address>:/data/ts_tutorial/p_amoA_FunGene9.5_isolates_assign/iTOL_output <path to Xmoa_sequences>
```
````

I believe we need the `-r` flag. Also `~` won't work with Powershell, being more general with `<path to Xmoa_sequences>` is more robust. students should have created the `Xmoa_sequences` as part of acquiring data.